roles/download/defaults/main.yml

```diff
@@ -10,7 +10,7 @@ kube_version: v1.4.3
 etcd_version: v3.0.6
 #TODO(mattymo): Move calico versions to roles/network_plugins/calico/defaults
 # after migration to container download
-calico_version: v0.22.0
+calico_version: v1.0.0-beta
```

Comment:
@mattymo in that case shouldn't we leave a possibility to deploy without TLS? Golang Calico (1.x.x) is still beta, but with this change users won't be able to deploy stable python-based (0.x.x) calico anymore. I propose to update defaults when v1.0.0 is released.